python/mxnet/ndarray/numpy/_op.py

```diff
@@ -732,3 +783,22 @@ def get_list(arrays):
 
     arrays = get_list(arrays)
     return _npi.stack(*arrays, axis=axis, out=out)
+
+
+@set_module('mxnet.ndarray.numpy')
+def slice_assign_scalar(lhs, value, begin, end, step, out=None):  # pylint: disable=too-many-arguments
```

Comment:
This not a numpy op and we do not need to define it here.

Reply:
Will resolve it. Thanks!